src/components/cardbuilder/Card/useCard.ts

```diff
@@ -81,6 +82,8 @@ function useCard({ item, cardOptions }: UseCardProps) {
         cardOptions.cardCssClass,
         cardOptions.cardClass,
         { 'card-hoverable': layoutManager.desktop },
+        { 'show-focus': layoutManager.tv },
+        { 'show-animation': layoutManager.tv && enableFocusTransform },
```

Comment:
If focus indication is broken in 10.9, this should be extracted as bugfix.

Reply:
This is the first time i used the card in stable router.

Reply:
Fix Search Field for Tv Mode #5610